### PR TITLE
convert - better feedback on exceptions as there's no feedback as to …

### PR DESCRIPTION
**Pull request by lnevay**
…which material specifically is the problem.

### CODE REVIEW

**Comment by ManuelHu** [comment]
could we use `raise _FLUKAError() from ex` here to avoid the `print()`? (Or any other type instead of _FLUKAError?)

**Comment by lnevay** [comment]
My intention was to catch the exception (so `Exception` as the base class is caught) and rethrow with an appended message but wasn't clear to me in python how to append the message, hence the print out.  If you know how to do that, feel free to implement.

Actually, if you look, FLUKAError has no implementation so I think better to use standard language exceptions and ones with no purpose.  Also, multiple types of exceptions could be thrown from that code so my intention wasn't to change the original exception type.

File: src/pyg4ometry/convert/fluka2g4materials.py [hunk 1]
```diff
@@ -166,12 +166,16 @@ def convertElement(self, name, flukaElement):
...
+        try:
+            if fractionType == "atomic":
+                self.convertAtomicFractionCompound(name, flukaCompound)
+            elif fractionType == "mass":
+                self.convertMassFractionCompound(name, flukaCompound)
+            elif fractionType == "volume":
+                self.convertVolumeFractionCompound(name, flukaCompound)
+        except Exception as ex:
+            print("Error in material definition: ", name)
+            raise ex
```
Comment (ManuelHu) [inline src/pyg4ometry/convert/fluka2g4materials.py:178]:
```suggestion
            msg = f"Error in material definition: {name}"
            raise _FLUKAError(msg) from ex
```

**Review by ManuelHu — COMMENTED** [review]
Then this should do the job :-)

**Pull request merged**